data/Telemetry/update.info

```diff
@@ -0,0 +1 @@
+https://gist.githubusercontent.com/FadeMind/2d58a7dc5c6ad33a4a45b7e8d2349fdd/raw/70dcb743f712c9ffe7feaa458c84360170051141/win.telem.merged.hosts
```

Comment:
@StevenBlack You have right. I will moving all gists hosted files to separated github repository and in another pull I send changes.